src/mesh/voxel_cloud.rs

```diff
@@ -34,6 +34,18 @@ use super::{primitive, tools, Face, Mesh};
 /// value outside of the block will cause the program to panic. Reading a value
 /// from beyond the bounds returns None, which is also a valid value even inside
 /// the block.
+///
+/// `volume_value_range` is an interval defining which values of the scalar
```

Comment:
Let's preface this comment with something like: "In this struct's method parameters, ...". This is what I meant by "You may have to adjust the comment to fit naturally in the new place." in my original review comment.